field/src/polynomial/mod.rs

```diff
@@ -186,11 +186,20 @@ impl<F: Field> PolynomialCoeffs<F> {
         poly
     }
 
-    /// Removes leading zero coefficients.
+    /// Removes any zero coefficients from the side associated with the highest exponents.
```

Comment:
nit: Hmm I find "coefficients from the side associated with the highest exponents" less clear than "leading coefficients". I think "leading coefficient" is well-established polynomial terminology, so I don't think there should be confusion about which side of the vector representation we're removing zeros from.

Reply:
Sounds good; I agree it was clunky wording. I was just thinking some readers might interpret "leading" as the front of the `Vec`, but hopefully not.